### ISSUE TITLE
*tsequenceset_make_gaps* is not implemented

### ISSUE THREAD

**Issue by satriabw** [MobilityDB/PyMEOS#50]
In MobilityDB 1.1, function *ttypeSequenceSetGaps* is used to define a gaps in the trajectory dataset and can be useful for data cleaning or defining trajectory. However, the implementation is missing from PyMEOS. Ideally PyMeos implements *TSequenceSetGap* and call *tsequenceset_make_gaps* to implement this functionality.

**Comment by Diviloper** [comment]
Indeed, creating `TSequenceSet`s from instants using the gaps functionality is not available in `pymeos`. Since the function is exposed by the `pymeos_cffi` library, adding it to `pymeos` should be straightforward (adding the method to the class and adding the proper tests).
If you are down to make a PR, I'll be more than happy to review it and merge it. Otherwise, I'll try to do it when I have a bit of time.
On the meanwhile though, note that you could still use this functionality, since you can call the `pymeos_cffi` function with the MEOS instant objects (`_inner` member of any PyMEOS object) and then passing the result MEOS object to the PyMEOS class constructor (`_inner` parameter)
